src/qibosoq/rfsoc_server.py

```diff
@@ -127,7 +133,7 @@ def handle(self):
         try:
             data = self.receive_command()
             results = execute_program(data, self.server.qick_soc)
-        except Exception as exception:  # pylint: disable=bare-except, broad-exception-caught
+        except Exception as exception:  # pylint: disable=W0612,W0718
```

Comment:
Btw, you're giving a name to the `exception`, but then you're not using it (I guess because `traceback` already knows about it).

Maybe you could drop the `as exception` part.

Reply:
Yes I think I can remove it... I was probably following some reference on how to log exceptions (following it blindly)